workflows/index-generation/generate_accession2taxid.py

```diff
@@ -4,143 +4,92 @@
 The full database contains billions of entries, but only ~15% of those are found in either NR or NT databases.
 Therefore, the full NCBI accession2taxid database is subsetted to include only the relevant entries and then used
 to map accessions to taxonomic IDs.
-Finally, the taxonomic lineage for each read is computed using the
```

Comment:
it would be nice to document more of the new process here, like what are the inputs/ outputs and data flow through the new program? Hopefully that's not too much work. Or link the confluence page if there is one